### DISCUSSION TITLE
Show a weird transparent window after upgrading at win 11

### DISCUSSION THREAD

**Discussion by ShannonZ** [AvaloniaUI/Avalonia#13443] [Q&A] [answered]
## Describe the bug
I can build and run the default template project given by the vs extension. Then a window containing Greeting Text showed. 
The default version of Avalonia is 11.0.2 in this project.
If upgrate all Avalonia relative packages  from 11.0.2 to 11.0.5 [using nuget manager] and rebuild the project, the rendered window become transparent and no text rendered.

## To Reproduce
1. Open VS and Create Avolonia C# Project targeting Desktop
2. Build and Run;
3. Upgrade all packages with newer version available.
4. Rebuild and Run, then see error

## Screenshots
![image](https://github.com/AvaloniaUI/Avalonia/assets/7865691/2cc4c566-0b39-4011-b18f-7c8efd6843d0)

## Environment 

- OS: Windows 11 Home Edition
- Avalonia-Version: 11.0.5, VS2022 17.7.6

**Comment by rabbitism** [comment]
You should include FluentTheme. 
![image](https://github.com/AvaloniaUI/Avalonia/assets/14807942/ea4819a1-82db-4cb6-bd7a-0708845bc83c)

**Comment by ShannonZ** [comment]
> You should include FluentTheme. ![image](https://user-images.githubusercontent.com/14807942/279266009-ea4819a1-82db-4cb6-bd7a-0708845bc83c.png)

`    <Application.Styles>
        <FluentTheme />
    </Application.Styles>`
    
The above code-snippet already exists in App.axaml

**Comment by maxkatz6** [comment]
Please, add logs. And ideally a minimal project, it's not a typical issue.

**Comment by ShannonZ** [comment]
> Please, add logs. And ideally a minimal project, it's not a typical issue.
[vs output.txt](https://github.com/AvaloniaUI/Avalonia/files/13212856/vs.output.txt)

I did not change any part the source code. Just create a desktop Avalonia C# Project and upgrade all upgradable nuget packages. Then I see the error.

**Comment by thevortexcloud** [comment]
Does it work if you update the templates and then create the project? EG

`dotnet new update`
`dotnet new avalonia.app`

**Comment by timunie** [comment, marked as answer]
Still probably a version mismatch so please upload your proj or show the Version of all Avalonia related libs of all proj in sln. Thx.

**Reply by ShannonZ** [reply]
Yes. It's all because of that I missed to upgrade the version of Avalonia.Desktop in the second project.

**Comment by maxkatz6** [comment]
> [Visual]Exception when rendering: 'System.TypeLoadException: Method 'PushRenderOptions' in type 'Avalonia.Skia.DrawingContextImpl' from assembly 'Avalonia.Skia, Version=11.0.2.0, Culture=neutral, PublicKeyToken=c8d484a7012f9a8b' does not have an implementation.

Yes, it is a version mismatch.

**Reply by ShannonZ** [reply]
So how to solve the problem?

**Reply by ShannonZ** [reply]
All works fine after upgrade Avalonia.Desktop to version 11.0.5.

**Comment by ShannonZ** [comment]
[AvaApp.zip](https://github.com/AvaloniaUI/Avalonia/files/13213419/AvaApp.zip)
Here is a minimal reproduced demo.

**Reply by maxkatz6** [reply]
You have not updated dependencies in the second project.

**Comment by ShannonZ** [comment]
Great thx to all of you.

**Comment by cemalgulbeyaz** [comment]
Yes i had same problem and it fixed after i upgrade Avalonia.Desktop nuget from Desktop project.
